web/themes/custom/pineapple/sass/layouts/nodes/_author.scss

```diff
@@ -0,0 +1,32 @@
+.author-body {
+  .cover-image {
+    height: 350px;
+    margin: -$unit -50% 0 -50%;
+    background-repeat: no-repeat;
+    background-position: 50%;
+  }
+
+  .author-info-container {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: $unit *4;
+    padding: $unit $unit*4;
+    padding-left: 0;
+    background-color: $light-gray;
+
+    .profile-image {
+      width: 100%;
+      margin-top: -15%;
+
+      img {
+        border-radius: 50%;
+        width: 100%;
+        object-fit: cover;
+      }
+    }
```

Comment:
On mobile the profile image should be much bigger than it is, at least 50% width at a minimum like it is on the full recipe page and the image should also be centralised in the middle of the screen horizontally, and vertically the image should look as it does on desktop